src/ofxVideoRecorder.cpp

```diff
@@ -470,6 +483,28 @@ void ofxVideoRecorder::close()
 
 }
 
+bool ofxVideoRecorder::hasVideoError()
+{
+    if (videoThread.bNotifyError) {
+        ofLogError("ofxVideoRecorder::addFrame()") << ofGetTimestampString("%H:%M:%S:%i") << " - Notify video error!.";
```

Comment:
This log seems redundant since the video writer thread already logged when the error occurred. No reason to log when you asked if there was an error.